sound/soc/sof/intel/hda-dsp.c

```diff
@@ -355,6 +363,11 @@ static int hda_resume(struct snd_sof_dev *sdev)
 	snd_sof_pci_update_bits(sdev, PCI_TCSEL, 0x07, 0);
 
 #if IS_ENABLED(CONFIG_SND_SOC_SOF_HDA)
+	if (runtime) {
+		/* read STATESTS before controller reset */
+		status = snd_hdac_chip_readw(bus, STATESTS);
```

Comment:
isn't this valid for the case CONFIG_SND_SOC_SOF_HDA_AUDIO_CODEC only?

SOF_HDA means HDMI only.

Reply:
yes, It is not for HDMI